# generated by patch-package 6.4.10
#
# command:
#   npx patch-package @firebase/auth --create-issue
#
# declared package:
#   @firebase/auth: ^1.5.1
#
diff --git a/node_modules/@firebase/auth/dist/esm2017/index-dd468b12.js b/node_modules/@firebase/auth/dist/esm2017/index-dd468b12.js
index 5ef851c..3a555cb 100644
--- a/node_modules/@firebase/auth/dist/esm2017/index-dd468b12.js
+++ b/node_modules/@firebase/auth/dist/esm2017/index-dd468b12.js
@@ -10047,7 +10047,7 @@ class BrowserPopupRedirectResolver {
     }
     get _shouldInitProactively() {
         // Mobile browsers and Safari need to optimistically initialize
-        return _isMobileBrowser() || _isSafari() || _isIOS();
+        return false;
     }
 }
 /**
